LSLib/LS/PackageCommon.cs

```diff
@@ -413,6 +410,9 @@ public void EnumerateFiles(Package package, string rootPath, string currentPath)
 
         public void CreatePackage(string packagePath, string inputPath, uint version = Package.CurrentVersion, CompressionMethod compression = CompressionMethod.None, bool fastCompression = true)
         {
+            FileManager.TryToCreateDirectory(packagePath);
+            FileManager.TryToCreateDirectory(inputPath);
```

Comment:
The input path should always exist, since we're creating a package from it, so only the first `TryToCreateDirectory` call is necessary.

Reply:
Fixed but not committed yet.